includes/class-wc-stripe-helper.php

```diff
@@ -1592,15 +1597,58 @@ public static function is_webhook_url( $url, $webhook_url = '' ) {
 			return true;
 		}
 
-		$webhook_url_parts = wp_parse_url( $url );
-		$url_parts         = wp_parse_url( $webhook_url );
+		$url_parts         = wp_parse_url( $url );
+		$webhook_url_parts = wp_parse_url( $webhook_url );
 
 		$url_host     = $url_parts['host'] ?? '';
 		$url_path     = $url_parts['path'] ?? '';
+		$url_query    = $url_parts['query'] ?? '';
 		$webhook_host = $webhook_url_parts['host'] ?? '';
 		$webhook_path = $webhook_url_parts['path'] ?? '';
+		$webhook_query = $webhook_url_parts['query'] ?? '';
 
-		return $url_host === $webhook_host && $url_path === $webhook_path;
+		if ( $url_host !== $webhook_host || $url_path !== $webhook_path ) {
+			return false;
+		}
+
+		// If the supplied URL has an empty query string, we will treat it as a webhook URL for the plugin,
+		// as we're guessing that it was created manually in the long-distant past when webhook
+		// management was all manual.
+		if ( '' === $url_query ) {
+			return true;
+		}
```

Comment:
This URL without a query param could be intentionally added too. However, in the reports, we haven't seen URLs created without any meaningful query params. So let's keep it this way and revisit if we get any complaints.